tools/reviewer/job/tasks/BUILD

```diff
@@ -0,0 +1,179 @@
+package(default_visibility = ["//visibility:public"])
+
+proto_library(
+    name = "submitter_proto",
+    srcs = ["submitter.proto"],
+)
+
+java_library(
+    name = "task_interface",
+    srcs = [
+        "Task.java",
+    ],
+    deps = [
+        "//common",
+        "//common:dagger_with_annotation_processor",
+        "//common/firestore",
+        "//common/flags",
+        "//common/repo",
+        "//third_party/maven/com/google/auth:google_auth_library_credentials",
+        "//third_party/maven/com/google/auth:google_auth_library_oauth2_http",
+        "//third_party/maven/com/google/flogger",
+        "//third_party/maven/com/google/flogger:flogger_system_backend",
+        "//third_party/maven/io/grpc:grpc_services",
+        "//third_party/maven/javax/inject:javax_inject",
+        "//tools/reviewer:reviewer_java_proto",
+        "//tools/reviewer:reviewer_registry_java_proto",
+        "//tools/reviewer/aa:aa_module",
+        "//tools/reviewer/aa/commands",
+        "//tools/reviewer/local_server",
+        "//tools/reviewer/local_server:local_http_gateway",
+        "//tools/reviewer/local_server/service:code_review_service",
+    ],
+)
+
+java_library(
+    name = "task_executor",
+    srcs = [
+        "TaskExecutor.java",
+    ],
+    deps = [
+        ":reviewer_metadata_updater_task",
+        ":task_interface",
+        "//common",
```

Comment:
@oferb it seems that this part of `deps` is repeated pretty often across this `BUILD` file. Do you think it's worth separating out into a variable or a separate `java_library` with `exports`?

Reply:
Actually, most of it is not needed, but it will get removed anyway once we have automated BUILD files, so I'm being `Lazy<>` :-P